main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -666,7 +668,8 @@ object Resolver {
             case NamedAst.DefOrSig.Sig(sig) => visitApplySig(app, sig, exps, region, innerLoc, outerLoc)
           }
 
-        case app@NamedAst.Expression.Apply(_, _, _) => visitApply(app, region)
+        case app@NamedAst.Expression.Apply(_, _, _) =>
```

Comment:
Remove app@

Reply:
This was already here. It's needed for the function call to `visitApply` :)